target/linux/bcm27xx/modules/other.mk

```
@@ -53,3 +53,63 @@ define KernelPackage/smi-bcm2835-dev/description
 endef
 
 $(eval $(call KernelPackage,smi-bcm2835-dev))
+
+
+define KernelPackage/rp1-firmware
+  TITLE:=RP1 Firmware Driver
+  KCONFIG:= \
+           CONFIG_FIRMWARE_RP1 \
+           CONFIG_MBOX_RP1
+  FILES:=$(LINUX_DIR)/drivers/firmware/rp1.ko
+  DEPENDS:=@!TARGET_bcm27xx_bcm(2708||2709||2710||2711) @TARGET_bcm27xx
```

Please write the full depends like like:
```suggestion
  DEPENDS:=@!TARGET_bcm27xx_bcm2708 @!TARGET_bcm27xx_bcm2709 @!TARGET_bcm27xx_bcm2710 @!TARGET_bcm27xx_bcm2711  @TARGET_bcm27xx
```
Maybe just making it depend on `@!TARGET_bcm27xx_bcm2712` is better.

---

@hauke - I trying your suggestion of the full depends now.  As to your 2nd suggestion, `@!TARGET_bcm27xx_bcm2712`, would make it build for 8,9,10, and 11 but we want it to only build for 12.